### DISCUSSION TITLE
Last share submitted time

### DISCUSSION THREAD

**Discussion by riksarchen** [coinfoundry/miningcore#957] [Ideas]
When browsing the database for other purpose I saw time when share was submitted was available. 

Any plans on adding that info to api?

**Comment by oliverw** [comment]
The database contains potentially tens of thousands of shares. What do you have in mind? :)

**Reply by riksarchen** [reply]
Just a timestamp of when last share was received. Would be nice to show on dashboard and could also be used for worker offline notification in future updates
